src/main/java/com/couchbase/lite/JavaContext.java

```diff
@@ -47,10 +51,7 @@ public SQLiteStorageEngineFactory getSQLiteStorageEngineFactory() {
 
     public File getRootDirectory() {
         String rootDirectoryPath = System.getProperty("user.dir");
-        File rootDirectory = new File(rootDirectoryPath);
-        rootDirectory = new File(rootDirectory, "data/data/com.couchbase.lite.test/files");
-
-        return rootDirectory;
+        return new File(rootDirectoryPath, "data/data/com.couchbase.lite.test/files");
```

Comment:
> Is it <USER HOME DIR>/data/data/com.couchbase.lite.test/files ?

 Yes.

> Is it expected path?

 It has been like this since the project was built. I didn't change anything. Another option would be current folder. If you wanna change, we can discuss.